templates/4_crystallicum_dark_templates/postbit/postbit_classic.html

```diff
@@ -1,45 +1,94 @@
-{$ignore_bit}
-<a name="pid{$post['pid']}" id="pid{$post['pid']}"></a>
-<div class="post classic {$unapproved_shade}" style="{$post_visibility}" id="post_{$post['pid']}">
-<div class="post_flexible">
-<div class="post_author scaleimages">
-	<center><strong><span class="largetext">{$post['profilelink']}</span></strong> {$post['onlinestatus']}</center>
-	{$post['useravatar']}<br>
-	<div class="author_information">
-			
-			<span class="smalltext">
-				{$post['usertitle']}<br />
-				{$post['userstars']}
-				{$post['groupimage']}
-			</span>
-	</div>
-	<div class="author_statistics">
-		{$post['user_details']}
-	</div>
-</div>
-<div class="post_content">
+<script>
+	document.addEventListener('DOMContentLoaded', function() {
+	  // Get the author avatar element
+	  const authorAvatar = document.querySelector('.author_avatar');
+  
+	  // Get the post author element
+	  const postAuthor = document.querySelector('.post_author');
+  
+	  // Check if the elements exist
+	  if (authorAvatar && postAuthor) {
+		// Get the height of the author avatar
+		const avatarHeight = authorAvatar.offsetHeight;
+  
+		// Set the min-height of the post author
+		postAuthor.style.minHeight = (avatarHeight + 10) + 'px';
+	  }
+	});
+  </script>
+  
+  <!--- Mechanika --->
+  <script>
+	document.addEventListener('DOMContentLoaded', function() {
```

Comment:
I'd really like us not to end up having hundreds of events registered for DOMContentLoaded. Why not just make it a function and call said function in onClick event for specific elements? That's something that can be made via template itself

Reply:
Done